paas_app_charmer/_gunicorn/logrotate.py

```diff
@@ -0,0 +1,60 @@
+#!/bin/python3
+
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Rotate Gunicorn logs.
+
+It will be injected into the container and must not dependencies on non-standard libraries.
+"""
+import argparse
+import datetime
+import glob
+import os
+import subprocess  # nosec B404
```

Comment:
Perhaps add a short explanation as to why security considerations regardig subprocess are not relevant in this module.

Reply:
Yes, indeed, updated, thanks!